scapy/tools/packet_viewer/viewlayer/views/main_view.py

```diff
@@ -34,17 +35,17 @@ def _create_format_string(columns):
             format_string += "{" + name + ":<" + str(length) + "}"
         return format_string
 
-    def __init__(self, socket, columns, _get_group, _get_bytes_for_analysis, basecls, **kwargs):
-        self.basecls = socket.basecls if hasattr(socket, "basecls") else basecls
-
-        self.columns = [("TIME", 20, lambda p: p.time), ("LENGTH", 7, lambda p: len(p))]
+    def __init__(self, socket, columns=None, _get_group=len,
+                 _get_data=lambda p: bytearray(bytes(p)),
```

Comment:
Eine Frage, warum wird hier `_get_data` und `_get_group` uebergeben. Diese Argumente werden nicht verwendet. Warum sind die Argumente in dieser Funktion?

Reply:
`get_data` und `get_group` werden noch nirgends verwendet. Wir wissen allerdings schon, dass wir die mal brauchen. Daher gibt es die schon und müssen nur noch verwendet werden, sobald wir sie brauchen.